.rubocop.yml

```diff
@@ -7,7 +7,7 @@ Metrics/CyclomaticComplexity:
 
 
 Metrics/LineLength:
-  Max: 120
+  Enabled: false
```

Comment:
This number was ridiculously high already anyways. I think we are disciplined enough to not need adult supervision here, so I'm going to disable this check. It interferes with Cucumber step definitions, which can become quite lengthy if written with descriptive variable names.